src/ngraph/opsets/opset0.hpp

```diff
@@ -0,0 +1,29 @@
+//*****************************************************************************
+// Copyright 2017-2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#pragma once
+
+#include "ngraph/ops.hpp"
+
+namespace ngraph
+{
+    namespace opset0
+    {
+#define NGRAPH_OP(a, b) using b::a;
+#include "ngraph/opsets/opset0_tbl.hpp"
+#undef NGRAPH_OP
+    }
+}
```

Comment:
This is one of my idea, I created `opset0.hpp` file like for opset1 to enable include operators from opset0.